nodeList.json

```diff
@@ -790,13 +813,23 @@
       "name": "E-Mesh North York GW",
       "latitude": 43.77966238349766,
       "longitude":-79.34211684973752,
-      "cardinalDirection": "West",
+      "cardinalDirection": "Omni",
       "status":"active",
       "floor": 3,
       "IPV6Address":"north-york-gw.darkdrgn2k.h.e-mesh.net",
```

Comment:
Can we have the IPv6 address here? Context: There's a missing IPv6 format prop in the schema that will be added. This value will fail the spec.

Reply:
Agreed, IP only in this field pls

Reply:
Host Names would have reduced the number of PRs required when nodes are rebuilt. 
However moved to IPv6 addy.